app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -958,6 +993,54 @@ class DefaultSessionControlControllerTest {
         }
     }
 
+    @Test
+    fun `WHEN handleOpenInPrivateTabClicked is called WITH a TopSite#Provided site THEN Event#TopSiteOpenContileInPrivateTab is reported`() {
+        val topSite = TopSite.Provided(
+            id = 1L,
+            title = "Mozilla",
+            url = "mozilla.org",
+            clickUrl = "",
+            imageUrl = "",
+            impressionUrl = "",
+            createdAt = 0
+        )
+        createController().handleOpenInPrivateTabClicked(topSite)
+
+        verify {
+            metrics.track(Event.TopSiteOpenContileInPrivateTab)
+        }
+    }
+
+    @Test
+    fun `WHEN handleOpenInPrivateTabClicked is called WITH TopSite#Default or TopSite#Pinned or TopSite#Frecent site THEN Event#TopSiteOpenInPrivateTab is reported`() {
```

Comment:
```suggestion
    fun `WHEN handleOpenInPrivateTabClicked is called with Default or Pinned or Frecent top site THEN TopSiteOpenInPrivateTab event is reported`() {
```

Perhaps a simplification to the #